llvm/test/SYCLLowerIR/lower_kernel_props.ll

```diff
@@ -33,7 +33,8 @@ define weak_odr dso_local spir_kernel void @__large_grf_kernel1() !sycl_explicit
 
 ; -- This kernel calls the marker function directly
 define weak_odr dso_local spir_kernel void @__large_grf_kernel2() #0 !sycl_explicit_simd !0 !intel_reqd_sub_group_size !1 {
-; CHECK: {{.*}} spir_kernel void @__large_grf_kernel2() #0
+; CHECK: {{.*}} spir_kernel void @__large_grf_kernel2() #0 {{.*}} !RegisterAllocMode ![[MetadataArg:[0-9]+]]
```

Comment:
How is this metadata preserved during SPIR-V translation? Do we have an extension already for it?

Reply:
@AlexeySachkov Ah sorry thought you were out. No, we have this: https://github.com/KhronosGroup/SPIRV-LLVM-Translator/commit/2439639ed8e572c372d0db8e8c10c5fc80e61c53

Planning on doing an extension in the future

Reply:
Thanks for the info. But I wouldn't recommend to expect stable replies from me for the next ~1.5 weeks :)

Reply:
You are a madman